domdiv/draw.py

```diff
@@ -25,19 +26,21 @@ def __init__(self):
         self.canvas = None
 
     def registerFonts(self):
-        try:
-            dirn = os.path.join(self.options.data_path, 'fonts')
+        dirn = os.path.join(self.options.data_path, 'fonts')
+        regularMpath = os.path.join(dirn, 'MinionPro-Regular.ttf')
+        boldMpath = os.path.join(dirn, 'MinionPro-Bold.ttf')
+        obliqueMpath = os.path.join(dirn, 'MinionPro-Oblique.ttf')
```

Comment:
This was a change based on the names downloaded from FontsGeek but it appears they've renamed to Italic I guess(?)  I will change it from `MinionPro-Oblique` back to `MinionPro-It`